test/unit/query-generator/arithmetic-query.test.ts

```diff
@@ -8,6 +8,86 @@ describe('QueryGenerator#arithmeticQuery', () => {
     firstName: DataTypes.STRING,
   }, { timestamps: false });
 
+  it('should use the plus operator', async () => {
+    const sql = queryGenerator.arithmeticQuery(
+      '+', 'myTable', {}, { foo: literal('bar') }, {}, {},
```

Comment:
arithmeticQuery really needs a good redesign, these options are incomprehensible (unrelated to this PR of course)

Reply:
Do you want me to make a TODO for that in the definition?

Reply:
No we'll get to it when we migrate it to TS I think